jscomp/test/list_test.ml

```diff
@@ -1,4 +1,4 @@
-
+let a = [1;2;3;4];;
```

Comment:
There is no reason for the `;;` to be at the end of the line here, especially as the rest of the file does not follow this format as well)?